packages/obonode/obojobo-chunks-list/viewer-component.js

```diff
@@ -27,6 +27,26 @@ const addItemToList = (ul, li, lis) => {
 	return lis.push(li)
 }
 
+const pickColorFromText = (node, indent) => {
+	let bulletColor
+	let textStyles
+
+	try {
+		// needs try/catch because some nodes don't have a text element
+		if (indent === 1) {
+			textStyles = node.children[0].text.styleList.styles
+			bulletColor = textStyles[textStyles.length - 1].data.text
```

Comment:
`node.children[0].text` is an instance of StyleableText, and it has some internals you can use to grab the styles on the text there. Grabbing the last item from `textStyles` is dangerous because there's no guarantee that the last item is color.

Instead, you can do this - `node.children[0].text.styleList.getStyleComparisonsForRange(0, 1, StyleType.COLOR)` - That will return an object that details where that style is, like this:

```js
{
    "after": [],
    "before": [],
    "enscapsulatedBy": [],
    "contains": [
        {
            "type": "color",
            "data": {
                "text": "#990000"
            },
            "start": 0,
            "end": 3
        }
    ],
    "left": [],
    "right": []
}
```

You'll want to look at `contains`. That means that the text from index 0 to 1 contains the color style. In which case, you can dig into `contains[0].data.text` to get the color.